src/components/layout/NavBar.less

```diff
@@ -115,7 +114,7 @@
 }
 
 .nav-bar__logo {
-  padding: 8px 0;
+  padding-top: 8px;
```

Comment:
i'm not sure about this because i'm in favor of keeping an equal padding on top and buttom. Maybe when it shrinks into media width we'd better add some boarders around those nav buttons to clearly indicate the padding vs the button area?

sorry just nitpicking about the details. i wish we have a designer 😥

Reply:
then i can keep the original line 👍 and i can override it in gitops.css if needed. it looks like most commons are overriding this anyway :/